lib/features/mailbox_dashboard/presentation/mailbox_dashboard_view.dart

```diff
@@ -5,16 +5,49 @@ import 'package:get/get.dart';
 import 'package:tmail_ui_user/features/email/presentation/email_view.dart';
 import 'package:tmail_ui_user/features/mailbox/presentation/mailbox_view.dart';
 import 'package:tmail_ui_user/features/mailbox_dashboard/presentation/base_mailbox_dashboard_view.dart';
+import 'package:tmail_ui_user/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart';
 import 'package:tmail_ui_user/features/mailbox_dashboard/presentation/model/dashboard_routes.dart';
 import 'package:tmail_ui_user/features/search/presentation/search_email_view.dart';
 import 'package:tmail_ui_user/features/thread/presentation/thread_view.dart';
 
 class MailboxDashBoardView extends BaseMailboxDashBoardView {
-
   MailboxDashBoardView({Key? key}) : super(key: key);
 
+  final mailBoxDashBoardController = Get.find<MailboxDashBoardController>();
```

Comment:
No need to `Get.find`. you can directly use the `controller` object which is an instant of MailboxDashBoardController